Settings-default.toml

```diff
@@ -33,7 +33,7 @@ path = "upload"
 [logging]
 # Minimum log level. Can be one of error, warn, info, debug, trace
 # or a more detailed spec. See https://docs.rs/flexi_logger/0.17.1/flexi_logger/struct.LogSpecification.html.
-log_spec = "info"
+log_spec = "debug"
```

Comment:
We should not change the default here.

Reply:
8fb81dbed4ec76c5ed14d5584083f7938818d7fe